### DISCUSSION TITLE
Displaying columns except following ones

### DISCUSSION THREAD

**Discussion by SanyaRus** [duckdb/duckdb#5789] [Ideas]
The idea is as follows: currently, there are the following ways to select columns from an SQL table for display:
1. SELECT * FROM table - displays the entire table
2. SELECT column1 FROM table - displays the selected columns of the table
I would also like to suggest another way to display:
3. SELECT * EXCEPT column 1 FROM table - displays the entire table EXCEPT column 1.
Rationale for the idea: there may be cases when there are many columns in the table, but it is not necessary to output all of it, and it is easier to specify which columns are NOT NEEDED than to specify which ones are needed for output.
The EXCEPT operator is already in the SQL language, but it is used in a slightly different way: all values in the table are output, except for those corresponding to some condition. I suggest using this operator to simply output columns OTHER than a certain one.

**Comment by SanyaRus** [comment]
For example, we have got table t1, that contains columns c1-c10.
We want to display all columns except c6.
Now we have to do following thing: SELECT c1, c2, c3, c4, c5, c7, c8, c9, 10 FROM t1;
But an easier thing is: SELECT * EXCEPT c8 FROM t1;

**Comment by lokax** [comment]
Try using EXCLUDE instead of EXCEPT

**Reply by SanyaRus** [reply]
Maybe EXCLUDE, no matter. But the principle must be: to display columns EXCEPT pointed ones.

**Reply by Alex-Monahan** [reply]
The exclude keyword already works today!
https://duckdb.org/docs/sql/expressions/star
